default.build

```diff
@@ -39,6 +41,10 @@
 			<arg value="/p:IncludeSymbols=&quot;True&quot;" />
 			<arg value="/p:IncludeSource=&quot;True&quot;" />
 			<arg value="/p:PackageOutputPath=&quot;${path::get-full-path(nuget.nupackages.dir)}&quot;" />
+			<!-- build.counter is not available by default. It needs to be exposed in Team City as system parameter system.build.counter with value %build.counter% -->
```

Comment:
As the `build.counter` and `build.number` are not available in the TC. Maybe it's better to set `VersionSuffix` directly instead?

Reply:
So release build will always generate dev versions? So how then generate actual release for actual releasing?
And then again dev builds may be published with wrong version  after release (it seems you don't see it as an issue though).

Reply:
Maybe as alternative build should be done without dev suffix. And then as additional step with dev suffix if dev publishing is enabled. But I don't like the idea that after release both branches will publish dev builds under the same version